### PR TITLE
Update korean translation #2

### PR DESCRIPTION
**Pull request by sujinleeme**
I updated Korean translation recently for our 2nd workshop. Please review it as [Django Girls Daejeon](https://www.djangogirls.org/daejeon) is going to use it this coming saturday 👍 🇰🇷

### CODE REVIEW

**Comment by patjouk** [comment]
Hi! Thanks for your hard and awesome work.
It's super short notice and bad timing for me (I'll be traveling all day tomorrow) so I'm not sure I'll have the time to review it before the workshop.

I saw that you change most https to http: can you put the https back? Thanks.

**Comment by sujinleeme** [comment]
Thanks for finding my some mistakes. I edited http to https that you said. Just relax and have a great vacation! :sunny:

File: ko/extend_your_application/README.md [hunk 1]
```diff
@@ -22,114 +22,111 @@
...
 
-첫 번째 게시물의 상세 **URL** 은 이렇게 나올 거에요. : http://127.0.0.1:8000/post/1/
+첫 게시물의 상세 페이지 **URL**이 http://127.0.0.1:8000/post/1/가 되게 만들 거에요.
 
-`blog/urls.py`파일에 URL을 만들어, 장고가 `post_detail`이란 *view* 로 보내, 전체 블로그 글이 보일 수 있게 만들어 봅시다. `url(r'^post/(?P<pk>[0-9]+)/$', views.post_detail, name='post_detail')` 코드를 `blog/urls.py` 파일에 추가하세요. 그러면 아래처럼 보일 거에요. :
+`blog/urls.py`파일에 URL을 만들어, 장고가 `post_detail` *뷰*로 보내, 게시글이 보일 수 있게 해봅시다. `url(r'^post/(?P<pk>[0-9]+)/$', views.post_detail, name='post_detail')`코드를 `blog/urls.py`파일에 추가하면 아래와 같을 보일 거에요.
 
 ```python
-from django.conf.urls import url
+from django.conf.urls import include, url
```
Comment (patjouk) [inline ko/extend_your_application/README.md]: You don't need to import include.

File: ko/extend_your_application/README.md [hunk 2]
```diff
@@ -142,34 +139,34 @@ urlpatterns = [
             </div>
         {% endif %}
         <h1>{{ post.title }}</h1>
-        <p>{{ post.text|linebreaksbr }}</p>
+        <p>{{ post.text|linebreaks }}</p>
```
Comment (patjouk) [inline ko/extend_your_application/README.md]: Can you revert those changes?

File: ko/extend_your_application/README.md [hunk 3]
```diff
@@ -142,34 +139,34 @@ urlpatterns = [
...
     </div>
 {% endblock %}
 ```
 
-다시 한 번 `base.html`을 확장해 봅시다. `content` 블록에서, 블로그 글의 published_date 출판일(존재한다면) 과 제목, 내용을 보이게 할 거에요. 그런데 제일 중요한 것을 얘기해봐야하지 않겠어요?
+다시 한 번 `base.html`을 확장한 거에요. `content`블록에서 블로그 글의 게시일, 제목과 내용을 보이게 만들었어요.
 
-{% raw %}`{% if ... %} ... {% endif %}` 는 무엇인가를 확인할 때 사용하는 템플릿 태그 입니다. (`if ... else ..` 를 __파이썬 들어가기__ 장에서 본 기억이나나요?) 우리는 post의 `출판 날짜(published_date)`가 비어있는지 아닌지 확인하고 싶어요. {% endraw %}
+가장 중요한 부분은 `{% if ... %} ... {% endif %}`라는 템플릿 태그인데, 내용이 있는지 확인할 때 사용합니다. (`if ... else ..`구문을 [**파이썬 들어가기**](http://tutorial.djangogirls.org/en/python_introduction/) 장에서 봤었죠?) `post`의 `게시일(published_date)`이 있는지, 없는지를 확인하는 거에요.
```
Comment (patjouk) [inline ko/extend_your_application/README.md]: Can you use https?

File: ko/how_the_internet_works/README.md [hunk 4]
```diff
@@ -1,53 +1,53 @@
 # 인터넷은 어떻게 작동할까요
 
-> 이번 장은 Jessica McKellar의 "How the internet works" 에서 영감을 받아 작성되었습니다. (http://web.mit.edu/jesstess/www/)
+> 이번 장은 Jessica McKellar의 글 ["How the internet works"](http://web.mit.edu/jesstess/www/)을 기초로 작성되었습니다. 
 
-우리는 매일 인터넷을 사용하고 있어요. 하지만 브라우저 주소창에 https://djangogirls.org 를 입력하고 `엔터`키를 누르면 어떤 일이 일어나는지 혹시 알고 있었나요?
+우리는 매일 인터넷을 사용합니다. 그러나 브라우저 주소창에 [http://djangogirls.org](http://djangogirls.org)를 입력하고 `엔터`키를 누르면 어떤 일이 일어나는지 알고 있나요?
```
Comment (patjouk) [inline ko/how_the_internet_works/README.md]: Can you use https?

File: ko/how_the_internet_works/README.md [hunk 5]
```diff
@@ -1,53 +1,53 @@
...
-조금 지저분해 보이죠? 사실 인터넷은 수 많은 기계들이 연결된(앞서 얘기한 *서버* 라는) 의 하나의 네트워크랍니다. 수백, 수천 대의 기계들이요! 긴, 아주 긴 케이블들이 지구 전체를 둘러싸고 있답니다! 해저 케이블 지도 사이트 (http://submarinecablemap.com/)에 가보면 케이블이 얼마나 복잡하게 연결되어있는지 알 수 있어요. 아래 사진은 웹사이트에서 가져온 스크린 샷입니다.
+조금 지저분해 보이죠? 사실 인터넷은 수많은 기계가 연결된(앞서 얘기한 *서버*라는) 의 하나의 네트워크랍니다. 수백, 수천 대의 기계들이요! 긴, 아주 긴 케이블들이 지구 전체를 둘러싸고 있답니다! [해저 케이블 지도 사이트](http://submarinecablemap.com/)에 가보면 케이블이 얼마나 복잡하게 연결되어있는지 알 수 있어요. 아래 사진은 웹사이트에서 가져온 스크린 캡처입니다.
 
 ![그림 1.2][2]
 
- [2]: images/internet_3.png
+  [2]: images/internet_3.png
 
-멋지지 않나요? 그러나 인터넷에 연결되어 있는 모든 기계들을 전부 연결하는 것은 불가능한 일입니다. 그래서 어떤 기계(예를 들어 https://djangogirls.org 이 저장되어 있는 것 같은)에 도달하기 위해서는 수없이 많은 다른 기계들을 통과해야 한답니다.
+멋지지 않나요? 그러나 인터넷에 연결된 모든 기계 전부 연결하는 것은 불가능한 일입니다. 그래서 특정한 기계(예를 들어 [http://djangogirls.org](http://djangogirls.org)가 저장되었다면)에 도달하기 위해서는 수없이 많은 다른 기계들을 통과해야 한답니다.
```
Comment (patjouk) [inline ko/how_the_internet_works/README.md]: Can you use https?

File: ko/how_the_internet_works/README.md [hunk 6]
```diff
@@ -1,53 +1,53 @@
...
 
 그림으로 보면 다음과 같습니다.
 
 ![그림 1.3][3]
 
  [3]: images/internet_2.png
 
-주소창에 https://djangogirls.org 을 입력하는 건, "사랑하는 장고 걸즈 여러분, 저는 장고걸즈 웹사이트를 보고 싶어요. 저에게 그 웹 사이트를 보여주세요!"라고 편지를 써서 보내는 것과 같아요.
+주소창에 http://djangogirls.org을 입력하는 건, "사랑하는 장고걸즈 여러분, 저는 장고걸즈 웹사이트를 보고 싶어요. 저에게 그 웹 사이트를 보여주세요!"라고 편지를 써서 보내는 것과 같아요.
```
Comment (patjouk) [inline ko/how_the_internet_works/README.md]: Can you use https?

File: ko/html/README.md [hunk 7]
```diff
@@ -1,194 +1,192 @@
 # HTML 시작하기
 
-템플릿이 무엇인지 궁금하셨죠?
+템플릿은 어떤 일을 하는지 궁금했죠?
 
-템플릿이란 서로 다른 정보를 일정한 형태로 표시하기 위해 재사용 가능한 파일이예요. 예를 들면 편지를 쓸때도 템플릿을 사용할 수 있어요. 편지의 내용이나 수신인 주소는 달라져도 동일한 디자인, 레이아웃을 사용하는 경우가 있으니까요.
+템플릿이란 서로 다른 정보를 일정한 형태로 표시하기 위해 재사용 가능한 파일을 말해요. 예를 들면 편지에도 같은 템플릿을 사용할 수 있어요. 편지의 내용이나 수신인 주소는 달라져도 같은 디자인, 레이아웃을 사용하는 때도 있으니까요.
 
-장고의 템플릿 양식은 HTML을 사용합니다. (HTML은 첫 번째 장 **인터넷은 어떻게 작동할까요** 에서 들어봤죠)
+장고 템플릿 양식은 HTML을 사용합니다. (HTML은 첫 번째 장 [**인터넷은 어떻게 작동할까요**](http://tutorial.djangogirls.org/ko/how_the_internet_works/)에서 들어본 내용이죠)
```
Comment (patjouk) [inline ko/html/README.md]: Can you use https?

**Comment by sujinleeme** [comment]
Most of work finished last summer and I have noticed that original tutorial has been changed slightly. Thanks for your Review!

**Comment by olasitarska** [comment]
@sujinleeme Do you want me to keep this open until you find time to work on this again? :)

**Pull request closed without merging**